test/run_each.sh

```diff
@@ -45,10 +45,11 @@ runTest () {
 }
 
 # These are the scripts that will be loaded for every test
-commonScripts="vendor/jquery.min.js"
+commonScripts="vendor/jquery.min.js,shared/javascripts/*.js"
 
 # Each line below executes the scripts in order in the context of the browsers.
-runTest "$commonScripts,../../test/sample_test.js"
+runTest "$commonScripts,../scripts/background_scripts/first_run.js,../../test/first_run.js"
```

Comment:
You should be able to put all these tests onto the same call to runTest. The reason privly-applications have so many different calls to runTest results from the fact that most of the code base was written before we had a good solution for unit testing. Running all the tests at once should work the same as running them separately, you just have to make certain that the code doesn't pollute the global namespace or clobber other functions.